### PR TITLE
Adding geom_ribbon in the list of geoms

### LINKED ISSUE #5779: Add geom_ribbon to geom list
`geom_ribbon` looks like a useful geom that we need https://ggplot2.tidyverse.org/reference/geom_ribbon.html. Can it be added to our list of geoms?

### PR DESCRIPTION
**Pull request by maxwellfundi**
@dannyparsons I have quickly added this. Please have look for any more additions you might like.

Fixes #5779

### CODE REVIEW

File: instat/ucrGeom.vb [hunk 1]
```diff
@@ -1217,20 +1217,29 @@ Public Class ucrGeom
...
-        'clsgeom_ribbon.AddAesParameter("colour")
-        'clsgeom_ribbon.AddAesParameter("linetype")
-        'clsgeom_ribbon.AddAesParameter("size")
+        clsgeom_ribbon.strGeomName = "geom_ribbon"
+        'mandatory aes are s or y or ymax or y min 
+        clsgeom_ribbon.AddAesParameter("x")
+        clsgeom_ribbon.AddAesParameter("y")
+        clsgeom_ribbon.AddAesParameter("ymax")
+        clsgeom_ribbon.AddAesParameter("ymin")
+        'optional
```
Comment (Wycklife) [inline instat/ucrGeom.vb:1226]: Why was this part left

**Comment by lloyddewit** [comment]
@maxwellfundi There's a small comment from @Wycklife still open. Once that's resolved, is this PR ready for test and peer review? thanks

**Comment by lloyddewit** [comment]
@maxwellfundi There's a small comment from @Wycklife still open. Once that's resolved, is this PR ready for test and peer review? thanks

**Comment by lloyddewit** [comment]
@maxwellfundi is this ready for test and peer review? thanks

**Comment by lloyddewit** [comment]
@maxwellfundi please could you clarify the status? thanks

File: instat/ucrGeom.vb (hunk 1 above)
Comment (maxwellfundi) [reply instat/ucrGeom.vb:1226]: @Wycklife by left you mean leaving the code for optional in ama?

Comment (maxwellfundi) [reply instat/ucrGeom.vb:1226]: The optional part .wants the aesthetic parameters are not mandatory to produce the plot

**Comment by lloyddewit** [comment]
@rdstern It would be good to close this PR. Please could you test and approve? thanks

**Comment by maxwellfundi** [comment]
@lloyddewit  Yeah, this has been here for a while. It is ready from my side. I added all that was needed. @Ivanluv @Wycklife  could you review this?

**Comment by Ivanluv** [comment]
@lloyddewit  am reviewing this

File: instat/ucrGeom.vb [hunk 2]
```diff
@@ -1217,20 +1217,29 @@ Public Class ucrGeom
...
-        'clsgeom_ribbon.AddAesParameter("x", bIsMandatory:=TRUE)
-        'clsgeom_ribbon.AddAesParameter("ymax", bIsMandatory:=TRUE)
-        'clsgeom_ribbon.AddAesParameter("ymin", bIsMandatory:=TRUE)
-        ''optional
-        'clsgeom_ribbon.AddAesParameter("alpha")
-        'clsgeom_ribbon.AddAesParameter("colour")
-        'clsgeom_ribbon.AddAesParameter("linetype")
-        'clsgeom_ribbon.AddAesParameter("size")
+        clsgeom_ribbon.strGeomName = "geom_ribbon"
+        'mandatory aes are s or y or ymax or y min
```
Comment (Ivanluv) [inline instat/ucrGeom.vb:1221]: A minor question,should the s  in the comment be x?

**Comment by Ivanluv** [comment]
@lloyddewit  this looks correct from my end. @rdstern  could  check if the parameters and aesthetics of the geom are  fine

**Comment by lloyddewit** [comment]
@rdstern Please could you test/approve?  thanks

**Pull request merged**